doc/api-use-case-health-organization.md

```diff
@@ -0,0 +1,276 @@
+    --data-urlencode "occurred_min=2015-01-01T00:00:00.000Z" \
+    --data-urlencode "record_type=0e222fc7-fee3-454e-a2c8-3eda5b9ef922" \
+    --data-urlencode 'jsonb={"person":{"Name":{"_rule_type":"containment_multiple","pattern":"John Doe"}, "Hospital":{"_rule_type":"containment_multiple","pattern":"Test Hospital"}}}'
+```
+
+
+## Updating a record within DRIVER
+
+In the previous example, a query was constructed for finding a specific person. It is also possible to modify the data found in DRIVER by submitting `PATCH` queries. For example, if "John Doe" in the previous example was incorrectly marked as sustaining a `Fatal` injury, but it was actually a `Serious` injury, the record may be altered via a `PATCH` request.
+
+In order to submit a `PATCH` request, the uuid of the record is needed. The previous response indicates that the uuid of this record is `a2b5586f-f77c-4fc0-bc3d-5c35167a6c54`, therefore we must submit a `PATCH` request to: "http://DRIVERURL/api/records/a2b5586f-f77c-4fc0-bc3d-5c35167a6c54". The request must contain the full json object retrieved in the original request, along with any desired modifications. Since in this example we only want to change the injury status of "John Doe" to `Serious`, only that one modification will be made and the `curl` command will look like:
+
+```
+curl -H "Authorization: Token TOKEN" \
+     -H "Content-Type:application/json" \
+     -X PATCH \
+     "http://DRIVERURL/api/records/a2b5586f-f77c-4fc0-bc3d-5c35167a6c54/" \
+     --data '{"uuid":"a2b5586f-f77c-4fc0-bc3d-5c35167a6c54","data":{"photo":[],"vehicle":[],"incidentDetails":{"Description":"","Num passenger casualties":"","Num driver casualties":"","Main cause":"Vehicle defect","Num pedestrian casualties":"","_localId":"364ddb5f-50fc-4c3e-91fb-1fde47d437ad","Severity":"Fatal","Collision type":"Right angle","Num vehicles":""},"person":[{"License number":"7777777","Name":"John Doe","Age":"","vehicle":"","Sex":"Male","Address":"123 Test St.","Injury":"Serious","Hospital":"Test Hospital","_localId":"2457f168-3ab7-4a43-a7e6-5a7a1b144277"},{"License number":"8888888","Name":"Jane Doe","Age":"","vehicle":"","Sex":"Female","Address":"234 Test St.","Injury":"Not injured","Hospital":"","_localId":"922ed1f0-b08b-49e9-8861-8e28ac9ecd2b"}]},"created":"2016-05-02T18:38:10.579710Z","modified":"2016-05-03T03:28:32.379533Z","occurred_from":"2015-01-02T06:33:13.947000Z","occurred_to":"2015-01-02T06:33:13.947000Z","geom":{"type":"Point","coordinates":[120.82582890987396,14.858803025925443]},"location_text":"Sumapa Ligas Road, Sumapang Matanda, Bulacan, Central Luzon, 3000, Philippines","city":null,"city_district":null,"county":null,"neighborhood":null,"road":"Sumapa Ligas Road","state":"Bulacan","weather":"clear-day","light":"dawn","archived":false,"schema":"fada1575-4f61-4575-bc8d-8b260ef93c31"}'
+```
+
+If the `PATCH` was successful, the updated record will be returned in the response, and it should contain the updates that have been made.
```

Comment:
There could also be a mention of status codes here as a way to determine update success.